File: Iniciante/1933.py
lines1 = list(map(int, input().split()))

c1 = lines1[0]
c2 = lines1[1]

if (c1 == c2):
  c3 = c1
elif (c1 > c2):
  c3 = c1
else:
  c3 = c2

print(c3)

File: Iniciante/1933.js
let lines1 = lines.shift().split(' ');

let c1 = Number(lines1.shift());
let c2 = Number(lines1.shift());
let c3;

if (c1 == c2) {
  c3 = c1;
} else if (c1 > c2) {
  c3 = c1;
} else {
  c3 = c2;
}

console.log(c3);
